e2etest/bgptests/infra_setup.go

```diff
@@ -81,11 +287,34 @@ func InfraSetup(ipv4Addresses, ipv6Addresses []string, externalContainers string
 			BGPPort:  179,
 			Password: "ibgp-test",
 		},
-		Network:  containersNetwork,
+		Network:  "host",
 		HostIPv4: hostIPv4,
 		HostIPv6: hostIPv6,
 	}
-	ibgpMultiHopContainerConfig := frrcontainer.Config{
+	return res
+}
+
+func frrContainersConfigs(ipv4Addresses, ipv6Addresses []string) map[string]frrcontainer.Config {
```

Comment:
naming is hard :P

Reply:
and, given this will return the configuration for when we run external containers, I am not sure that's the right name. And I was skeptic to call them "defaultVRFContainersConfig", which is what they are, because I didn't want to give vrf too much weight when we don't use it. So all in all I don't think the name is that bad. Could be defaultFRRContainersConfig but even that doesn't sound too appealing. WDYT?

Reply:
I thought that `externalContainersConfigs` is for the external containers?
from what I'm seeing this is called only by `KindnetContainersSetup` :sweat_smile: